apps/studio/components/interfaces/Database/Triggers/TriggersList/TriggerList.tsx

```diff
@@ -100,28 +97,24 @@ export const TriggerList = ({
       {_triggers.map((x) => (
         <TableRow key={x.id}>
           <TableCell className="space-x-2">
-            <Tooltip>
-              <TooltipTrigger
-                onClick={() => editTrigger(x)}
-                className="cursor-pointer text-foreground truncate max-w-48 inline-block"
-              >
-                {x.name}
-              </TooltipTrigger>
-              <TooltipContent side="bottom" align="center">
-                {x.name}
-              </TooltipContent>
-            </Tooltip>
+            <Button
+              type="text"
+              onClick={() => editTrigger(x)}
+              title={x.name}
+              className="text-link-table-cell text-left text-sm min-w-0 p-0 hover:bg-transparent font-medium max-w-48 title"
+            >
+              {x.name}
+            </Button>
```

Comment:
Tooltip felt a bit unnecessary. I added `title` (native tooltip as explained above) in the rare case that it is truncated.